api/MediaSource.json

```diff
@@ -501,7 +501,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "8"
+              "version_added": false
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/MediaSource/isTypeSupported is testing the wrong thing here, the original data is probably correct. unittest/unit/build.js even has unit tests for this API specifically, but still an instance is being tested. @vinyldarkscratch can you look into this?